euphonic/readers/castep.py

```diff
@@ -72,10 +73,7 @@ def read_phonon_dos_data(
             weights = np.concatenate((weights, [frequency_block.weight]))
             freqs = np.concatenate((freqs, [frequency_block.frequencies]))
             mode_grads = np.concatenate((mode_grads, [qmode_grad.squeeze()]))
-        line = f.readline()
-        if 'BEGIN DOS' not in line:
-            raise RuntimeError(
-                f'Expected "BEGIN DOS" in {filename}, got {line}')
+        f.readline()  # Skip line
```

Comment:
Slight change in functionality?

Reply:
Again, it's a specific warning for a failure state that seems very hard to reach. A file like this will still lead to an error, just a slightly less helpful one. But I don't know how you end up with such a file; non-DOS CASTEP outputs will blow up while reading numbers from the header section.